cli/commands/pods.go

```diff
@@ -24,29 +23,44 @@ func (cmd *podHandler) handleList(a *kingpin.Application, e *kingpin.ParseElemen
 }
 
 func (cmd *podHandler) handleStatus(a *kingpin.Application, e *kingpin.ParseElement, c *kingpin.ParseContext) error {
-	endpointPath := "v1/pod/status"
+	var endpointPath string
 	if len(cmd.PodName) > 0 {
```

Comment:
Would it make sense to introduce a local variable here that captures this condition since it is used again later?

Reply:
Done